src/main/java/com/indiealexh/MainVerticle.java

```diff
@@ -0,0 +1,243 @@
+package com.indiealexh;
+
+import com.indiealexh.models.FireData;
+import com.indiealexh.models.GeoLocation;
+import com.indiealexh.models.StateFireData;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.CompositeFuture;
+import io.vertx.core.Future;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpServer;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.client.HttpResponse;
+import io.vertx.ext.web.client.WebClient;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MainVerticle extends AbstractVerticle {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MainVerticle.class);
+
+    private HttpClient httpClient;
+
+
+    public FireData FireData;
+
+    @Override
+    public void start(Future<Void> startFuture) {
+        httpClient = vertx.createHttpClient();
+        Future<Void> steps = updateDatabase().compose(v -> startHttpServer());
+        steps.setHandler(ar -> {
+            if (ar.succeeded()) {
+                LOGGER.info("Succeeded boot up OnFire");
+                startFuture.complete();
+                // 30 Min repeated timer
+                vertx.setPeriodic(1800000, id -> {
+                    LOGGER.info("Updating database");
+                    updateDatabase();
+                });
+            } else {
+                LOGGER.info("Failed boot up");
+                startFuture.fail(ar.cause());
+            }
+        });
+    }
+
+
+
+    private Future<String> downloadXmlFireData() {
+        Future<String> futureWebResponse = Future.future();
+        WebClient webClient = WebClient.wrap(httpClient);
+
+        webClient
+                .get(443, "rmgsc.cr.usgs.gov", "/outgoing/GeoMAC/current_year_fire_data/KMLS/ActiveFirePerimeters.kml")
```

Comment:
It's going to be hardcoded one way or another, but I suggest moving the URL to a constant on top of the class or even a config file to make it easier to change.